### PR TITLE
Add email verified

### PR DESCRIPTION
**Pull request by dorshay6**
## ℹ️ Change description:
Nothing here

<details><summary><sub>templates<sub></summary>
<p>
  
<sub>[templates: [bug](?expand=1&template=bug.md), [feature](?expand=1&template=feature.md)]</sub>

</p>
</details>

### CODE REVIEW

File: examples/client/Locomotion/src/I18n/en.json [hunk 1]
```diff
@@ -58,6 +58,13 @@
                 "resendCodeText": "Didn't receive an SMS? ",
                 "resendCodeButton": "Resend"
             },
+            "emailCode": {
+                "title": "Verification",
+                "text": "Please enter code",
+                "subText": "A code has been sent to the mail:  \n{{email}}.",
+                "resendCodeText": "Didn't receive an EMAIL? ",
```
Comment (shirmagen) [inline examples/client/Locomotion/src/I18n/en.json]: sure email in caps?

Comment (dorshay6) [reply examples/client/Locomotion/src/I18n/en.json]: still wating for text from Ital

Comment (dorshay6) [reply examples/client/Locomotion/src/I18n/en.json]: But changed for now :)

File: examples/client/Locomotion/src/context/user/index.tsx [hunk 2]
```diff
@@ -106,6 +114,15 @@ const UserContextProvider = ({ children }: { children: any }) => {
     }
   };
 
+  const onEmailVert = async (code: string) => {
+    const vertResponse = await emailVerify({
+      email: user?.email,
+      code,
+    });
+
+    return vertResponse.status === 'OK';
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/context/user/index.tsx:123]: Let's work with status codes

File: examples/client/Locomotion/src/pages/Profile/EmailCode.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+const Code = () => {
+  const { user, onEmailVert, updateUserFromServer } = useContext(UserContext);
+  const navigation = useNavigation();
+  const [code, setCode] = useState('');
+  const [showErrorText, setShowErrorText] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const onVertCodeChange = (value) => {
+    setShowErrorText(false);
+    if (value.length === 4) {
```
Comment (avivbarniv) [inline examples/client/Locomotion/src/pages/Profile/EmailCode.js:23]: extract 4 to const

**Review by avivbarniv — APPROVED** [review]

File: examples/client/Locomotion/src/context/user/index.tsx (hunk 2 above)
Comment (dorshay6) [reply examples/client/Locomotion/src/context/user/index.tsx:123]: Using the status code for this does not make sense

Comment (dorshay6) [reply examples/client/Locomotion/src/context/user/index.tsx:123]: This is not client error like bad password is not 400

Comment (dorshay6) [reply examples/client/Locomotion/src/context/user/index.tsx:123]: firebase do it as well

**Pull request merged**